### PR TITLE
Add/Update monthly controls [Oct 23]. Closes #711

### LINKED ISSUE #711: Add/Update monthly controls [Oct 23]
**Is your feature request related to a problem? Please describe.**

- [ ] Add additional service controls

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Priyanka-Chatterjee-2000**
### Checklist
- [ ] Issue(s) linked

### CODE REVIEW

File: conformance_pack/iam.sp [hunk 1]
```diff
@@ -2158,3 +2166,75 @@ query "iam_policy_custom_no_permissive_role_assumption" {
...
+    )
+    select
+      p.arn as resource,
+      case
+        when bad.arn is null then 'ok'
+        else 'alarm'
+      end status,
+      p.name || ' contains ' || coalesce(bad.statements_num,0)  ||
+        ' statements that allow action "*" on resource "*".' as reason
+      ${replace(local.common_dimensions_qualifier_global_sql, "__QUALIFIER__", "p.")}
```
Comment (misraved) [inline conformance_pack/iam.sp:2235]: Does the resource support tags? If yes, I think we should add the tag dimensions as well.

File: conformance_pack/iam.sp [hunk 2]
```diff
@@ -2158,3 +2166,75 @@ query "iam_policy_custom_no_permissive_role_assumption" {
...
+        inline_policies_std,
+        name,
+        account_id,
+        region,
+        _ctx,
+        'iam_group' as type
+      from
+        aws_iam_group
+    ),
+    bad_policies as (
```
Comment (misraved) [inline conformance_pack/iam.sp:2206]: Can we rename this CTE to full_administrative_privilege_policies?

File: conformance_pack/iam.sp [hunk 3]
```diff
@@ -663,6 +663,14 @@ control "iam_policy_custom_no_permissive_role_assumption" {
   tags = local.conformance_pack_iam_common_tags
 }
 
+control "iam_inline_policy_no_administrative_privileges" {
+  title       = "IAM inline policy should not have administrative privileges"
+  description = "Ensure that no inline IAM policies exist which allow administrative privileges."
```
Comment (misraved) [inline conformance_pack/iam.sp:668]:
```suggestion
  description = "Ensure that no inline IAM policies exist that allow administrative privileges."
```

File: conformance_pack/athena.sp [hunk 4]
```diff
@@ -29,3 +37,21 @@ query "athena_workgroup_encryption_at_rest_enabled" {
...
+  sql = <<-EOQ
+    select
+      name as resource,
+      case
+        when enforce_workgroup_configuration then 'ok'
+        else 'alarm'
+      end as status,
+      case
+        when enforce_workgroup_configuration then name || ' enforce workgroup configuration enabled.'
+        else name || ' enforce workgroup configuration disabled.'
```
Comment (misraved) [inline conformance_pack/athena.sp:51]:
```suggestion
        when enforce_workgroup_configuration then name || ' has enforce workgroup configuration enabled.'
        else name || ' has enforce workgroup configuration disabled.'
```

**Review by misraved — COMMENTED** [review]
@Priyanka-Chatterjee-2000 please take a look at the review comments. Thanks!!

File: conformance_pack/iam.sp (hunk 1 above)
Comment (Priyanka-Chatterjee-2000) [reply conformance_pack/iam.sp:2235]: For aws_iam_role and aws_iam_user they support tagging but aws_iam_group does not support tagging and in this query, we are doing a union of the three tables, that is the reason we cannot use tags here.

**Review by madhushreeray30 — APPROVED** [review]

**Pull request merged**